geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -1303,6 +1303,11 @@ private void initializePartitionedRegion() {
     this.prRoot = PartitionedRegionHelper.getPRRoot(getCache());
   }
 
+  // Used for testing purposes
```

Comment:
should not introduce this method. Should use prRoot = PartitionedRegionHelper.getPRRoot(getCache()); instead.

Reply:
Created getPRRoot to allow for mocking within tests.
Static helper class would not allow this.